### PR TITLE
Creating Blog Post of gRPCConf2024

### PR DESCRIPTION
**Pull request by nikitaTay**
Screenshot 2024-11-18 1 17 12 PM
Screenshot 2024-11-18 1 01 12 PM
Screenshot 2024-11-18 1 01 01 PM
Screenshot 2024-11-18 12 59 46 PM
Screenshot 2024-11-18 12 59 19 PM
Changes for creating a Blog Post of gRPCConf2024

### CODE REVIEW

File: content/en/blog/grpconf-2024-videos.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Highlights in gRPConf 2024:Customer Showcase, Developer Engagement, Birds of Feathers Discussions and more.
+date: 2024-07-03
```
Comment (YifeiZhuang) [inline content/en/blog/grpconf-2024-videos.md:3]: why this date? change to the date that this blog post is published?

File: content/en/blog/grpconf-2024-videos.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+We loved hosting gRPConf 2024 last month in Sunnyvale, California. Thank you to
```
Comment (YifeiZhuang) [inline content/en/blog/grpconf-2024-videos.md:6]: it's not last month. "This year" or "This summer".

**Review by YifeiZhuang — APPROVED** [review]
minor comments

File: content/en/blog/grpconf-2024-videos.md (hunk 1 above)
Comment (nikitaTay) [reply content/en/blog/grpconf-2024-videos.md]: done

File: content/en/blog/grpconf-2024-videos.md (hunk 2 above)
Comment (nikitaTay) [reply content/en/blog/grpconf-2024-videos.md]: done

**Review by sreenithi — APPROVED** [review]

**Pull request merged**